includes/commands/class-mu-migration-export.php

```diff
@@ -51,6 +51,22 @@ public static function getCSVHeaders() {
 		return $headers;
 	}
 
+	public function meta( $args = array(), $site_data = array() ) {
```

Comment:
This should be a protected method, otherwise it will be exposed as a sub-command of the export command.

Reply:
Also can you please add some PHPDoc to this mehtod?